jekyll/_cci2/server/operator/installing-server-behind-a-proxy.adoc

```diff
@@ -16,32 +16,47 @@ toc::[]
 
 [#installation-and-configuration]
 == Installation and configuration
-There are two stages to installing CircleCI server behind a proxy. First, at the point of installation, the proxy addresses need to be specified, along with any addresses that should not be behind the proxy.
 
-[#installing-behind-a-proxy]
-=== Installing behind a proxy
-The installation process is described in detail in the https://circleci.com/docs/2.0/server-3-install/[CircleCI Server v4.x Installation guide]. Both proxy and non-proxy addresses should be supplied using the arguments described https://kots.io/kotsadm/installing/online-install/#proxies[here]. The installation command should be in the format:
+To enable the proxy, first toggle `proxy.enabled` to `"1"`.
 
-[source,bash]
-----
-kubectl kots install circleci-server --http-proxy <my-http-proxy-uri> --https-proxy <my-https-proxy> --no-proxy <my-no-proxy-list>
-----
+Next, fill out `proxy.http.host` and `proxy.https.host`, along with their associated port. These values can be the same but they both need to be configured.
 
-[#configuring-your-proxy]
-=== Configuring your proxy
-Once you have installed server and accessed the management console, there are some fields that need to be completed in the configuration section, as shown in the screenshot below. These fields will not be automatically populated, so the same proxy and no-proxy addresses you supplied during installation will need to be supplied here. If your proxy requires authentication in the form of a username and password, check the *HTTP Proxy authenticated* option to add the credentials.
+In order to configure authentication, you will need to configure `proxy.http.auth.enabled` and `proxy.https.auth.enabled` as `"1"`. You will also need to configure the respective username and password for both HTTP and HTTPS.
 
-.CircleCI Server v4.x Configuring a Proxy
-image::proxy-settings.png[Screenshot showing settings available to configure a proxy]
+Finally, configure the `no_proxy` hosts and subnets. This should include localhost, your GitHub Enterprise host (optional), and the hostname of your CircleCI installation (see limitations below for an explanation).
+
+```yaml
+proxy:
+  enabled: "1"
+  http:
+    host: proxy.example.internal
+    port: "3128"
+    auth:
+      enabled: "1"
+      username: <proxy-user>
+      password: <proxy-password>
+  https:
+    host: proxy.example.internal
+    port: "3128"
+    auth:
+      enabled: "1"
+      username: <proxy-user>
+      password: <proxy-password>
+  no_proxy:
+    - localhost
+    - 127.0.0.1
+    - github.example.internal
+    - circleci.example.internal
+    - <vpc-or-subnet-cidr>   # VPC or subnets to exclude from the proxy (optional)
```

Comment:
Should we make a suggestion to put nomad subnets in here, or are those covered by something else?